### PR TITLE
`ignorePartials` query to not parse partials

### LINKED ISSUE #95: Define partials at build-time/render-time
For compatibility reasons, I have a project using some bundled modules with mustache templates, where the partial "path" doesn't exactly match the module name/location. We were using Hogan.js and providing those partials with their compiled templates at rendertime, rather than compiletime. We're looking at upgrading the Mustache templates to Handlebars, but the old templates fail to compile with `handlebars-loader` because it fails to resolve the templates.

Is there any way to tell the loader to use a particular template for a given partial? Or failing that, ensure it doesn't error out and continue to provide those partials at rendertime?

### PR DESCRIPTION
**Pull request by mattberkowitz**
I believe this will resolve #95

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,6 +58,8 @@ module.exports = function(source) {
 
   var debug = query.debug;
 
+  var ignorePartials = !!query.ignorePartials
```
Comment (pcardune) [inline index.js]: I don't think this line is really necessary if you just use `query.ignorePartials` directly below.

**Comment by pcardune** [comment]
this mostly looks good to me. Can you update the CHANGELOG.md file as well?

**Comment by mAAdhaTTah** [comment]
Oh nice, this would be helpful. If I'm not mistaken, I'm able to pass in the partials at render-time already, correct?

**Comment by mattberkowitz** [comment]
@pcardune I've made the requested changes

@mAAdhaTTah Yes, this option will cause the loader to not attempt to load any partial references. I use it in my codebase where we define all of our partials in code

**Comment by pcardune** [comment]
Thanks! I'll put out a new release with this change in the next few days.

**Comment by mattberkowitz** [comment]
@pcardune any chance you could do a release to get this on npm?

**Pull request merged**